impeller/playground/backend/vulkan/playground_impl_vk.h

```diff
@@ -41,8 +43,6 @@ class PlaygroundImplVK final : public PlaygroundImpl {
   PlaygroundImplVK(const PlaygroundImplVK&) = delete;
 
   PlaygroundImplVK& operator=(const PlaygroundImplVK&) = delete;
-
-  static void InitGlobalVulkanInstance();
```

Comment:
If this is being removed, then also remove the declaration of `global_instance_` in `playground_impl_vk.h`